src/test/java/net/minecraftforge/debug/item/ItemPreventPickupTest.java

```diff
@@ -20,22 +24,46 @@
 import net.minecraftforge.event.entity.item.ItemTossEvent;
 import net.minecraftforge.event.entity.player.ItemTooltipEvent;
 import net.minecraftforge.eventbus.api.Event;
-import net.minecraftforge.eventbus.api.EventPriority;
-import net.minecraftforge.eventbus.api.IEventBus;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
-import net.minecraftforge.items.pickup.ItemPickupQuery;
+import net.minecraftforge.items.pickup.ItemPickupPredicate;
 import net.minecraftforge.items.pickup.ItemPickupReasons;
 import net.minecraftforge.registries.DeferredRegister;
 import net.minecraftforge.registries.ForgeRegistries;
 import net.minecraftforge.registries.RegistryObject;
 
 import java.util.List;
 
+/**
+ * The ItemPreventPickupTest is the test mod for testing the {@link ItemPickupReasons} system.
+ * <p>When this mod is enabled the following is enabled:
+ *
+ * <p>This mod adds a new item to the game {@code item_prevent_pickup_test:dummy}
+ * in which should only be picked up under the following circumstances
+ * <ul>
+ *     <li>If Player is sneaking</li>
+ *     <li>If Zombie is standing on a Stone block</li>
```

Comment:
It should be noted that the Zombie needs to have the capacity to pickup loot, as controlled by the `CanPickUpLoot` byte tag, with a 55% chance of being randomly enabled in spawned zombies. A zombie which can pickup loot is spawned by the following command: `/summon minecraft:zombie ~ ~ ~ {CanPickUpLoot:1b}`.